extensions/amp-analytics/amp-analytics.md

```diff
@@ -442,6 +443,7 @@ NOTE: There is a [known issue](https://github.com/ampproject/amphtml/issues/1089
 
 The `visibilitySpec` is a set of conditions and properties that can be applied to `visible` or `hidden` triggers to change when they fire. If multiple properties are specified, they must all be true in order for a request to fire. Configuration properties supported in `visibilitySpec` are:
   - `waitFor`: This property indicates that the visibility trigger should wait for a certain signal before tracking visibility. The supported values are `none`, `ini-load` and `render-start`. If `waitFor` is undefined, it is defaulted to [`ini-load`](#initial-load-trigger) when selector is specified, or to `none` otherwise.
+  - `reportWhen`: This property indicates that the visibility trigger should wait for a certain signal before sending the trigger. The supported values are the empty string, and `endOfFrame`.
```

Comment:
let's not say we support empty string. Only `endOfFrame` will be supported

Reply:
Done